ArduPlane/events.cpp

```diff
@@ -94,7 +94,10 @@ void Plane::failsafe_long_on_event(enum failsafe_state fstype, mode_reason_t rea
     case Mode::Number::QLOITER:
     case Mode::Number::QACRO:
     case Mode::Number::QAUTOTUNE:
-        set_mode(mode_qland, reason);
+        if (quadplane.options & QuadPlane::OPTION_FS_ACTION) {
+            set_mode(mode_qrtl, reason);
+           }
+        else set_mode(mode_qland, reason);
```

Comment:
This should be putting the action on a seperate line, inside of a `{` block, as per the style guide.

Reply:
done